edit text up until security considerations section (#86)
decentralized-identity/jwt-vc-presentation-profile: spec/spec.md

This document aims to provide a path to interoperability by standardizing the set of specifications that enable the presentation of JWT VCs between implementers. Future versions of this document will include details on issuance and wallet interoperability. Ultimately, this profile will define a standardized approach to Verifiable Credentials so that distributed developers, apps, and systems can share credentials through common means.

### Scope

#### Scope
This document is currently scoped for the presentation of VCs between the Self-Issued OP and the Verifier/RP, also known as the RP. The Self-Issued OP is a native mobile application. The following aspects of the presentation are covered:

This document is currently scoped for the presentation of VCs between the Wallet and the Verifier. The Wallet is a native mobile application. The following aspects are in scope:

- Data model
- Key management
- Transportation of VCs
- Query language
- Protocol to request presentation of VCs, including query language
- User authentication layer using Self-Issued ID Token
- Mechanism to establishing trust in the DID via Domain Linkage
- Identifiers of the entities
- Revocation of VCs

> The JWT VC Presentation Profile currently supports only one response mode, assuming that a Self-Issued OP is on a different device than the one on which the End-User has initiated a user interaction at the Verifier/RP, even if it is not.

Reviewer: we decided in the last meeting to keep this for v0.1 and then remove it for v1.0

> - Mechanism to the status of the VCs

Reviewer: i dont like the wording of this. why did you replace Revocation of VCs with this? Are we wanting to support other VC statuses?

> - Mechanism to the status of the VCs

Author: ```suggestion
- Revocation of VCs
```

    - Selective disclosure and unlinkability
    - Zero-Knowledge Proofs
    - Disputes
- Non-native Wallets like web applications, PWAs, etc.

> Note: Although selective disclosure and unlinkability are out of scope of this document, future versions will consider Selective Disclosure for JWTs ([[ref: SD-JWT]]), JSON Web Proofs ([[ref:JWP]]) and JSON Web Algorithms ([[ref:JWA]]) once they mature in IETF.

Reviewer: we haven't discussed this yet so feels a bit soon to add? i guess the language is vague enough?

- To bind an owner of a DID to a controller of a certain origin, a Well Known DID Configuration MUST be used as defined in [[ref: Well Known DID]].
- For Revocation of VCs, Status List 2021 as defined in [[ref: Status List 2021]] MUST be discovered using either DID Relative URLs stored in an HTTPS URL or ID Hub be used in combination with Identity Hubs as defined in [[def: Identity Hub (0.0.1 Predraft)]] (Decentralized Web Node v0.0.1 predraft).

This profile uses certain versions of specifications that have not yet reached final status: For more details see [Normative References](#normative-references) section.

  - First Implementer's Draft of Self-Issued OpenID Provider v2 specification
  - First Implementer's Draft of OpenID for Verifiable Prensetations specification
  - ID Hub specification published as a v0.0.1 predraft of [Decentralized Web Node](https://identity.foundation/decentralized-web-node/spec/). Throughout the document, the term ID Hub, rather than Decentralized Web Node, will be used to avoid confusion.

> This JWT VC Presentation Profile currently only supports response mode `direct_post`, sending Authorization Response as HTTP POST request, even if redirect would be possible since the End User is interacting with the Wallet and the the Verifier/RP on the same device.

Reviewer: ```suggestion
The JWT VC Presentation Profile currently only supports response mode `direct_post`, sending Authorization Response as HTTP POST request, even if redirect would be possible since the End User is interacting with the Wallet and the the Verifier/RP on the same device.
```

Reviewer: I have never heard of `direct_post` response mode. Is this a known term? link could be good?

Author: post was changes to direct_post, only a parameter name

> This JWT VC Presentation Profile currently only supports response mode `direct_post`, sending Authorization Response as HTTP POST request, even if redirect would be possible since the End User is interacting with the Wallet and the the Verifier/RP on the same device.

Author: ```suggestion
The JWT VC Presentation Profile currently only supports response mode `direct_post` defined in [[ref: OpenID4VP ID1]], sending Authorization Response as HTTP POST request, even if redirect would be possible since the End User is interacting with the Wallet and the the Verifier/RP on the same device.
```

> The JWT VC Presentation Profile currently only supports response mode `direct_post` defined in [[ref: OpenID4VP ID1]], sending Authorization Response as HTTP POST request, even if redirect would be possible since the End User is interacting with the Wallet and the the Verifier/RP on the same device.

Author: ```suggestion
The JWT VC Presentation Profile currently only supports response mode `direct_post` defined in [[ref: OpenID4VP ID1]], sending Authorization Response as HTTP POST request.
```
